scripts/template_upload_all.py

```diff
@@ -81,8 +81,6 @@ def template_name(image_link, image_ts, checksum_link, version=None):
             return "cfme-nightly-{}-{}".format(version, result[0][:-2])
         elif version:
             # proper build
-            if len(version) == 4:
-                version = version[:-1] + '0' + version[-1:]
```

Comment:
This was intentional, and AFAIK its driven by polarion templates for our builds being in the format: `x.y.z.a`, where `a` is always zero padded.

This is mirrored in other places in the scripts having to do with miq_version and polarion upload. If we want to remove this zero padding we'll have to make a more complete change.

Again, this all falls into the bucket of changes I'm trying to make across our tooling for modular template upload.

Reply:
I'll think over this once more and maybe try to make versions keep points.